src/gwt/src/org/rstudio/studio/client/application/events/EventBus.java

```diff
@@ -172,6 +178,11 @@ private final native void fireEventToSatellite(JavaScriptObject data,
       target.fireRStudioEventExternal(data, "");
    }-*/;
    
+   private void debugLogEvents(GwtEvent<?> event)
```

Comment:
I'm surprised Eclipse is happy with this apparently unused private method ... since it's a one-liner maybe just make that line the commented out bit?